caom2/caom2/plane.py

```diff
@@ -1149,15 +1149,24 @@ def bandpass_name(self, value):
         self._bandpass_name = value
 
     @property
-    @deprecated(version='CAOM2.4', reason='Replaced by energy_bands')
+    @deprecated(version='CAOM2.4',
+                reason='Replaced by energy_bands, gone in 2.5')
     def em_band(self):
         """ EM Band """
-        return None
+        result = None
+        if len(self._energy_bands) == 1:
+            result = next(iter(self._energy_bands)).value
```

Comment:
If len>2 we should probably err because someone used both energy_bands and em_band.

Reply:
Returned the implementation to 'do nothing', as that seems to be the only way to not interfere with the checksums.